gms/gossiper.cc

```diff
@@ -1739,15 +1739,16 @@ future<> gossiper::start_gossiping(int generation_nbr, std::map<application_stat
         }).then([this, generation] {
             logger.trace("gossip started with generation {}", generation);
             _enabled = true;
+            _abort_source = abort_source();
```

Comment:
Why is this line needed? _abort_source should be already constructed in the constructor by then.